client/pages/mockups.js

```diff
@@ -0,0 +1,999 @@
+						end
+						className={ ( { isActive } ) =>
+							`flex items-center gap-2 ${
+								isActive ? 'bg-sky-600' : 'hover:bg-slate-600'
+							} text-white font-bold py-2 px-4 rounded`
+						}
+					>
+						<svg
+							xmlns="http://www.w3.org/2000/svg"
+							fill="none"
+							viewBox="0 0 24 24"
+							strokeWidth={ 1.5 }
+							stroke="currentColor"
+							className="w-6 h-6"
+						>
+							<path
+								strokeLinecap="round"
+								strokeLinejoin="round"
+								d="M6.827 6.175A2.31 2.31 0 015.186 7.23c-.38.054-.757.112-1.134.175C2.999 7.58 2.25 8.507 2.25 9.574V18a2.25 2.25 0 002.25 2.25h15A2.25 2.25 0 0021.75 18V9.574c0-1.067-.75-1.994-1.802-2.169a47.865 47.865 0 00-1.134-.175 2.31 2.31 0 01-1.64-1.055l-.822-1.316a2.192 2.192 0 00-1.736-1.039 48.774 48.774 0 00-5.232 0 2.192 2.192 0 00-1.736 1.039l-.821 1.316z"
+							/>
+							<path
+								strokeLinecap="round"
+								strokeLinejoin="round"
+								d="M16.5 12.75a4.5 4.5 0 11-9 0 4.5 4.5 0 019 0zM18.75 10.5h.008v.008h-.008V10.5z"
+							/>
+						</svg>
+						New Screenshot
+					</NavLink>
+
+					<NavLink
+						to="/mockups/edit"
+						className={ ( { isActive } ) =>
+							`flex items-center gap-2 ${
+								isActive ? 'bg-sky-600' : 'hover:bg-slate-600'
+							} text-white font-bold py-2 px-4 rounded`
+						}
+					>
+						<svg
+							xmlns="http://www.w3.org/2000/svg"
+							fill="none"
+							viewBox="0 0 24 24"
+							strokeWidth={ 1.5 }
+							stroke="currentColor"
+							className="w-6 h-6"
+						>
+							<path
+								strokeLinecap="round"
+								strokeLinejoin="round"
+								d="M3.75 6A2.25 2.25 0 016 3.75h2.25A2.25 2.25 0 0110.5 6v2.25a2.25 2.25 0 01-2.25 2.25H6a2.25 2.25 0 01-2.25-2.25V6zM3.75 15.75A2.25 2.25 0 016 13.5h2.25a2.25 2.25 0 012.25 2.25V18a2.25 2.25 0 01-2.25 2.25H6A2.25 2.25 0 013.75 18v-2.25zM13.5 6a2.25 2.25 0 012.25-2.25H18A2.25 2.25 0 0120.25 6v2.25A2.25 2.25 0 0118 10.5h-2.25a2.25 2.25 0 01-2.25-2.25V6zM13.5 15.75a2.25 2.25 0 012.25-2.25H18a2.25 2.25 0 012.25 2.25V18A2.25 2.25 0 0118 20.25h-2.25A2.25 2.25 0 0113.5 18v-2.25z"
+							/>
+						</svg>
+						Browse Screenshots
+					</NavLink>
+				</div>
+			</nav>
+
+			<Outlet />
+		</div>
+	);
+}
+
+function Home() {
+	return (
+		<div className="p-4 text-center">
+			<div className="inline-flex gap-4 items-end p-8 bg-white rounded-md shadow-md text-left">
+				<label className="w-64">
+					<span class="block text-slate-600 text-sm mb-2">Site:</span>
+					<select className="block w-full rounded-md border-slate-300 focus:border-slate-400 focus:ring focus:ring-slate-600 focus:ring-opacity-10">
+						<option>WordPress.com</option>
+						<option>WordPress.com</option>
+					</select>
+				</label>
+
+				<label className="w-64">
+					<span class="block text-slate-600 text-sm mb-2">
+						Resolution:
+					</span>
+					<select className="block w-full rounded-md border-slate-300 focus:border-slate-400 focus:ring focus:ring-slate-600 focus:ring-opacity-10">
```

Comment:
Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.

Reply:
It's supposed to be transformed into a controlled field later when integrated as a component, so I think it's fine to leave it as is for the purpose of the mockup.